moveit_planners/chomp/chomp_motion_planner/include/chomp_motion_planner/multivariate_gaussian.h

```diff
@@ -63,32 +61,26 @@ class MultivariateGaussian
   Eigen::MatrixXd covariance_cholesky_; /**< Cholesky decomposition (LL^T) of the covariance */
 
   int size_;
-  boost::mt19937 rng_;
-  boost::normal_distribution<> normal_dist_;
-  boost::variate_generator<boost::mt19937, boost::normal_distribution<> > gaussian_;
+  std::mt19937 rng_;
+  std::normal_distribution<double> gaussian_;
 };
 
 //////////////////////// template function definitions follow //////////////////////////////
 
 template <typename Derived1, typename Derived2>
 MultivariateGaussian::MultivariateGaussian(const Eigen::MatrixBase<Derived1>& mean,
                                            const Eigen::MatrixBase<Derived2>& covariance)
-  : mean_(mean)
-  , covariance_(covariance)
-  , covariance_cholesky_(covariance_.llt().matrixL())
-  , rng_()
-  , normal_dist_(0.0, 1.0)
-  , gaussian_(rng_, normal_dist_)
+  : mean_(mean), covariance_(covariance), covariance_cholesky_(covariance_.llt().matrixL()), gaussian_(0.0, 1.0)
```

Comment:
It looks like you are no longer initializing `normal_dist_`. Are you sure that's OK? Or maybe `normal_dist_` is not needed and could be removed.

Reply:
Yeah I did remove `normal_dist_` from the rest of the function here. `std` doesn't need it, you can sample directly from the `std::normal_distribution` object by using the Mersenne twister object.